internal/cli/root.go

```diff
@@ -21,11 +21,6 @@ const rootShort = "Supercharge your development workflow."
 
 // Execute is the primary entrypoint of the CLI app.
 func Execute() {
-	// cfg contains tenant related information, e.g. `travel0-dev`,
```

Comment:
Removing this comment because it references a now irrelevant `AUTH0_API_KEY` env var and generally doesn't apply to this codebase anymore.